### PR TITLE
chore: update kubernetes api references

### PR DESCRIPTION
**Pull request by philoserf**
- update all apiVersion entries in templates
- bump verions

incidentals

- improve README
- use explicit versions
- use consistent YAML indention
- update docker base image

closes: jira/in-148
related: jira/in-138

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
 ## How to install on running Kubernetes cluster with `helm`
-Ensure that you have helm and [tiller](https://docs.helm.sh/using_helm/) installed. 
-### From our chart repository
-``` 
-helm repo add cnct https://charts.migrations.cnct.io
-helm repo update
-helm install cnct/eventrouter
-```  
+
+Ensure that you have helm and [tiller][5] installed. \#\#\# From our chart repository
```
Comment (DStorck) [inline README.md]: this looks strange when you preview it , not sure what the desired output is

**Comment by mikeln** [comment]
no, not really chicken and egg problem.  more of a don't do 2 changes at once.
The internal change to the created image should be separate.  That would include the change in the chart to read that image.

The other "bring up to date the templates" changes should not include that type of change.

File: charts/eventrouter/values.yaml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 name: eventrouter
 replicaCount: 1
-image: quay.io/samsung_cnct/eventrouter-container:0.2.1
+image: quay.io/samsung_cnct/eventrouter-container:0.3.0
 imagePullPolicy: IfNotPresent
```
Comment (mikeln) [inline charts/eventrouter/values.yaml]: Don't do this in this PR.
Version change should be a separate PR.

File: pipeline.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 type: chart
 envValues:
   - envVar: KUBECTL_VERSION
-    value: v1.9.2
+    value: v1.13.0
 rootfs:
-  - image: samsung_cnct/eventrouter-container
+  - image: samsung_cnct/eventrouter-container:0.3.0
     dockerContext: rootfs/eventrouter
```
Comment (mikeln) [inline pipeline.yaml]: Don't do this in this PR.
Version change should a separate PR.

File: rootfs/eventrouter/Dockerfile [hunk 4]
```diff
@@ -12,10 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.5
+FROM alpine:3.6
 LABEL MAINTAINER="Samsung CNCT OSS Maintainers <oss-maintainer@samsung-cnct.io>"
```
Comment (mikeln) [inline rootfs/eventrouter/Dockerfile]: This should in a separate PR.

File: README.md (hunk 1 above)
Comment (philoserf) [reply README.md]: Ah. An autoformat miss. I have a rewritten sentence to push.

File: charts/eventrouter/values.yaml (hunk 2 above)
Comment (philoserf) [reply charts/eventrouter/values.yaml]: ACK

**Comment by philoserf** [comment]
Okay, a consensus is in. Separate smaller PRs.
